test/unit/dummy-test.js

```diff
@@ -0,0 +1,7 @@
+import {expect} from '@loopback/testlab';
+
+describe('dummy test - remove me later', () => {
```

Comment:
What's the purpose of the dummy test here?

Reply:
CI thinks that having no tests found is a failure, so this is just a placeholder.

Reply:
In that case, I think we should comment on the test to describe the purpose of its presence and explicitly state to remove it once some tests are made.